Makefile

```diff
@@ -62,7 +62,7 @@ detect-crds-drift:
 
 clean:
 	@echo "cleaning up caches and output"
-	go clean -cache -testcache -r -x ./... 2>&1 >/dev/null
+	go clean -r -x ./... 2>&1 >/dev/null
```

Comment:
We still want to clean the cache for the tests. Any reason those two flags got removed?